scripts/funcotator/data_sources/gnomAD/createGnomadAlleleFreqTsv.json

```diff
@@ -0,0 +1,10 @@
+{
+  "CreateGnomadAlleleFreqTsv.gatk_docker": "broadinstitute/gatk:4.0.11.0",
+
+  "CreateGnomadAlleleFreqTsv.gnomAD_file": "gs://gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.vcf.bgz",
+  "CreateGnomadAlleleFreqTsv.out_file_name": "gnomad.genomes.r2.1.sites.alleleFreqs.tsv",
+
+  "CreateGnomadAlleleFreqTsv.mem_gb": "128",
+  "CreateGnomadAlleleFreqTsv.disk_space_gb": "16384",
```

Comment:
16 TB?!  Ouch....

Reply:
Yeah... I want to make sure it doesn't run out of memory.

I had it happen to me once, so I bumped it up to make sure it didn't happen again.